### DISCUSSION TITLE
SKU Economics Report

### DISCUSSION THREAD

**Discussion by kevnhowe** [amzn/selling-partner-api-models#3973] [Q&A] [answered]
Is the SKU Economics Report available via Seller Central available via the SP-API Reports API? I see a note which mentions the SP-API, but is vague: "Note: * denoted fields are not yet available in the downloadable report but are available via the reports SP-API."

**Comment by Javadebi** [comment]
Hello, did you find the report?

**Reply by ashish-atidiv** [reply]
Hi @kevnhowe @Javadebi 
Hello, did you find the report?

**Comment by ashish-atidiv** [comment, marked as answer]
I think it is available via the [Data Kiosk API ](https://docs.aws.amazon.com/prescriptive-guidance/latest/strategy-gen-ai-selling-partner-api/data-sp-api.html)

**Reply by kevnhowe** [reply]
This is correct. The current version of the [Data Kiosk API](https://developer-docs.amazon.com/sp-api/docs/data-kiosk-api-v2023-11-15-reference) contains an "analytics_economics_2024_3_15" graphql schema that contains both "economics" and "economics preview" queries. These queries are API versions of the "Selling and Economics and Fees" queries found on Seller Central Reports. I have not done a thorough comparison between the API and Seller Central versions, but what I've seen leads me to believe they retrieve and present the same data. You can build the Data Kiosk queries using the [Data Kiosk Schema Explorer](https://sellercentral.amazon.com/datakiosk-schema-explorer?schema=analytics_economics_2024_03_15). It is what I use to build my query strings to send in the request.

**Reply by ElephantandAntzDigitalSolutionsLLP** [reply]
Guys,
Have you worked with the Data Kiosk API ?? I am Facing some issues using it and would like to discuss it further

**Reply by ashish-atidiv** [reply]
Hi @ElephantandAntzDigitalSolutionsLLP 
Please share some more details.

**Reply by ElephantandAntzDigitalSolutionsLLP** [reply]
Hey @ashish-atidiv  please check this, I have details here https://github.com/amzn/selling-partner-api-models/discussions/4699
